### PR TITLE
Complete task DOM API

### PR DESCRIPTION
**Pull request by deveLabR**
### DOM API

[Demo](https://develabr.github.io/dom-js/) |
[Code base](https://github.com/deveLabR/dom-js)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by deveLabR** [comment]
I finished the self-check. And I hope it's okay)

File: submissions/develabr/dom-js/app.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  showMenu.classList.toggle('menu--open');
+});
+
+// Insert content
+
+const btns = document.querySelectorAll('.navbar-link');
+const main = document.querySelector('.main');
+
+for (const btn of btns) {
+  btn.addEventListener('click', function () {
```
Comment (madmaxWMFU) [inline submissions/develabr/dom-js/app.js:19]: Read about [event delegation](https://javascript.info/event-delegation)

File: submissions/develabr/dom-js/app.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      addContent(cityItem);
+    }
+  });
+}
+
+const addContent = (item) => {
+  main.innerHTML = `
+	<section class="main-section">
+	<header class="header-content">
+		<img class="emblem" src="${item.emblem}" alt="Герб ${item.name}" />
```
Comment (madmaxWMFU) [inline submissions/develabr/dom-js/app.js:32]: Read about [destruction](https://javascript.info/destructuring-assignment)

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good work. See some comments below.

File: submissions/develabr/dom-js/app.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  iconMenu.classList.toggle('icon--active');
+  showMenu.classList.toggle('menu--open');
+});
+
+// Insert content
+
+const listBtn = document.querySelector('.navbar-list');
+const main = document.querySelector('.main');
+
+let [kyiv, lviv, mykolaiv, odesa, ternopil, kharkiv, kherson, chernivtsi] =
```
Comment (madmaxWMFU) [inline submissions/develabr/dom-js/app.js:18]: Must be const. To get the first element use: `const [ kyiv ] = cities;`

File: submissions/develabr/dom-js/app.js [hunk 4]
```diff
@@ -12,18 +12,19 @@ iconMenu.addEventListener('click', function (e) {
...
-      addContent(cityItem);
-    }
-  });
-}
+let [kyiv, lviv, mykolaiv, odesa, ternopil, kharkiv, kherson, chernivtsi] =
+  cities;
+
+listBtn.onclick = function (event) {
+  let target = event.target;
+  if (target && target.classList.contains('navbar-link')) {
```
Comment (madmaxWMFU) [inline submissions/develabr/dom-js/app.js:23]: If you read about destruction, use it.
```javascript
listBtn.onclick = function ({ target }) {
  if (target.classList.contains('navbar-link')) {
```

File: submissions/develabr/dom-js/app.js [hunk 5]
```diff
@@ -12,18 +12,19 @@ iconMenu.addEventListener('click', function (e) {
...
+    const cityItem = cities.find(({ id }) => id === target.dataset.city);
+    addContent(cityItem);
+  }
+};
 
 const addContent = (item) => {
```
Comment (madmaxWMFU) [inline submissions/develabr/dom-js/app.js:29]:
```
const addContent = (city) => {
 const {sourse, founded, population, ........ } = city;`

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]

**Review by madmaxWMFU — APPROVED** [review]
Great, I approve of your work. Good luck with future tasks!

**Pull request merged**